less/icons.less

```diff
@@ -9,7 +9,7 @@
       margin-top: -25px;
       margin-left: -12px;
       min-width: 18px;
-      padding: 2px 3px;
+      padding: 3px 3px 2px 2px;
```

Comment:
Why is this necessary?

Reply:
It doesn't look centered in your screenshot.

Reply:
I wish we would use two overlaid elements, one which has the actual circle and the one with the actual number for the most precise alignment. We found that padding combination (which nudges the content to the centre) to be the "best" option.